docker/pandoc-lambda/function/old_pr1491/table_of_contents.lua

```diff
@@ -0,0 +1,147 @@
+toc_prefix = [[
```

Comment:
I pulled down the `table_of_contents.lua` that's on develop and ran diff: 
```
diff ~/Downloads/table_of_contents.lua docker/pandoc-lambda/function/old_pr1491/table_of_contents.lua
1d0
< 
54,56d52
<         <w:tabs>
<           <w:tab w:val="right" w:leader="dot" w:pos="9350"/>
<         </w:tabs>
64d59
<             <w:rStyle w:val="Hyperlink"/>
65a61
>             <w:iCs/>
```
https://github.com/harvard-lil/h2o/blob/develop/docker/pandoc-lambda/function/table_of_contents.lua#L54-L56 is present on develop, but not in `old_pr1491/table_of_contents.lua`; there's `<w:iCs/>` in `old_pr1491/table_of_contents.lua` but not on develop. 

Just checking, is that correct? I would have expected these files to be the same.

Reply:
Yep. Forgot I made style updates to this, too.